libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1634,8 +1634,8 @@ void AP_InertialSensor::_acal_save_calibrations()
 void AP_InertialSensor::_acal_event_failure()
 {
     for (uint8_t i=0; i<_accel_count; i++) {
-        _accel_offset[i].set_and_save(Vector3f(0,0,0));
-        _accel_scale[i].set_and_save(Vector3f(0,0,0));
+        _accel_offset[i].set(Vector3f(0,0,0));
```

Comment:
Why would this cause a crash on a second accel cal before? And if we are setting the parameters still why won't that still be the behavior?